recipes/default.rb

```diff
@@ -26,12 +24,20 @@
 end
 
 unless node['mremoteng']['shared_config_dir'].nil?
-  hosts = search(:node, 'name:*')
+  hosts = partial_search(:node, 
+                         "name:*",
+                         :keys => { 
+                           'hostname' => ['hostname'],
+                           'name' => ['name'],
+                           'os' => ['os'],
+                           'domain' => ['domain'],
+                           'env' => ['chef_environment']
```

Comment:
Why would you change the `chef_environment` key to just `env` here? Is there any reason to do that? If you leave it to be just `chef_environment` I think the code is more clear and less changes are needed.